### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Koliras**

### CODE REVIEW

**Review by AntonHerasymov — APPROVED** [review]
Good job!

File: src/inverseRobot.js [hunk 1]
```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
```
Comment (dffUqp) [inline src/inverseRobot.js:11]: I think we can make this shorter, let's get robot value in `for` loop and then use something like this:
image

File: src/inverseRobot.js [hunk 2]
```diff
@@ -7,7 +7,20 @@
...
+  const INVERSED_ROBOT = {};
```
Comment (dffUqp) [inline src/inverseRobot.js:12]: it's better to rename this variable to lowercase, basically we use upper + snake keys for static variables that won't be changed

learn more about javascript naming convention - [link](https://www.robinwieruch.de/javascript-naming-conventions/)

**Review by dffUqp — CHANGES_REQUESTED** [review]
Almost done:smiley:

**Review by BudnikOleksii — APPROVED** [review]
Nicely done👍

**Pull request closed without merging**